extensions/keycloak-authorization/runtime/src/main/java/io/quarkus/keycloak/pep/KeycloakPolicyEnforcerConfig.java

```diff
@@ -0,0 +1,192 @@
+        @ConfigItem
+        boolean httpMethodAsScope;
+
+        @ConfigGroup
+        public static class PathConfig {
+
+            /**
+             * The name of a resource on the server that is to be associated with a given path
+             */
+            @ConfigItem
+            Optional<String> name;
+
+            /**
+             * A URI relative to the application’s context path that should be protected by the policy enforcer
+             */
+            @ConfigItem
+            Optional<String> path;
+
+            /**
+             * The HTTP methods (for example, GET, POST, PATCH) to protect and how they are associated with the scopes for a
+             * given
+             * resource in the server
+             */
+            @ConfigItem
+            Map<String, MethodConfig> methods;
+
+            /**
+             * Specifies how policies are enforced
+             */
+            @DefaultConverter
+            @ConfigItem(defaultValue = "ENFORCING")
+            PolicyEnforcerConfig.EnforcementMode enforcementMode;
+
+            /**
+             * Defines a set of one or more claims that must be resolved and pushed to the Keycloak server in order to make
+             * these
+             * claims available to policies
+             */
+            @ConfigItem
+            ClaimInformationPointConfig claimInformationPoint;
+        }
+
+        @ConfigGroup
+        public static class MethodConfig {
+
+            /**
+             * The name of the HTTP method
+             */
+            @ConfigItem
+            String method;
+
+            /**
+             * An array of strings with the scopes associated with the method
+             */
+            @ConfigItem
+            List<String> scopes;
+
+            /**
+             * A string referencing the enforcement mode for the scopes associated with a method
+             */
+            @DefaultConverter
+            @ConfigItem(defaultValue = "ALL")
+            PolicyEnforcerConfig.ScopeEnforcementMode scopesEnforcementMode;
+        }
+
+        @ConfigGroup
+        public static class PathCacheConfig {
+
+            /**
+             * Defines the time in milliseconds when the entry should be expired
+             */
+            @ConfigItem(defaultValue = "1000")
+            int maxEntries = 1000;
+
+            /**
+             * Defines the limit of entries that should be kept in the cache
+             */
+            @ConfigItem(defaultValue = "30000")
+            long lifespan = 30000;
+        }
+
+        @ConfigGroup
+        public static class ClaimInformationPointConfig {
+
+            /**
+             *
+             */
+            @ConfigItem(name = ConfigItem.PARENT)
+            Map<String, Map<String, Map<String, String>>> complexConfig;
+
+            /**
+             *
+             */
+            @ConfigItem(name = ConfigItem.PARENT)
+            Map<String, Map<String, String>> simpleConfig;
+        }
+    }
+}
```

Comment:
I'm not sure how many of those properties are related to the KC policy enforcer configuration the user was trying to migrate. Please review and leave only those which are needed to support that policy-enforcer configuration; everything else can go later on to `quarkus-oidc` or `quarkus-oidc-uma` for example

Reply:
Except for the `userManagedAccess` all other properties are specific to KC policy enforcer.